src/main/java/net/forgecraft/services/ember/util/Util.java

```diff
@@ -0,0 +1,74 @@
+package net.forgecraft.services.ember.util;
+
+import com.electronwill.nightconfig.toml.TomlParser;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.databind.json.JsonMapper;
+import net.forgecraft.services.ember.Main;
+import net.forgecraft.services.ember.app.Services;
+import org.apache.hc.client5.http.impl.DefaultRedirectStrategy;
+import org.apache.hc.client5.http.impl.classic.CloseableHttpClient;
+import org.apache.hc.client5.http.impl.classic.HttpClients;
+
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+public class Util {
+
+    // TODO move to config
+    /**
+     * A list of known maven repositories to try and resolve maven artifacts against
+     */
+    @Deprecated
+    public static final List<String> KNOWN_MAVENS = List.of(
+            "https://api.modrinth.com/maven",
+            "https://cursemaven.com",
+            "https://maven.blamejared.com",
+            "https://modmaven.k-4u.nl",
+            "https://maven.saps.dev/releases",
+            "https://maven.saps.dev/snapshots",
+            "https://maven.nanite.dev/releases",
+            "https://maven.nanite.dev/snapshots",
+            "https://maven.fabricmc.net",
+            "https://maven.creeperhost.net",
+            "https://maven.minecraftforge.net",
+            "https://maven.neoforged.net/releases",
+            "https://repo.spongepowered.org/maven",
+            "https://maven.uuid.gg/releases"
+    );
+
+    //TODO might be wise to use a different http library that allows setting default headers such as User-Agent
```

Comment:
This could be done by using a wrapper RequestBuilder method that would return a prefilled builder with custom headers etc that can be used to build off of for new requests.

Reply:
that was a leftover comment, removed.